foreman.te

```diff
@@ -83,6 +83,20 @@ gen_tunable(passenger_can_connect_openstack, true)
 ## </desc>
 gen_tunable(passenger_can_spawn_ssh, true)
 
+## <desc>
+## <p>
+## Determine whether passenger can connect to itself (HTTPS) when preloading app.
+## </p>
+## </desc>
+gen_tunable(passenger_can_connect_http, true)
+
+## <desc>
+## <p>
+## Determine whether passenger can connect to puppet master (HTTPS) when preloading app.
```

Comment:
```the puppet master port``` and ```the app```. Also not sure if HTTPS here should be mentioned since it's really just a TCP connection that's allowed. What protocol they use isn't really checked by SELinux.

Reply:
You are right, in both cases port types are abstract things, they refer to 0-N port numbers. Changed.